GVRf/Framework/jni/engine/importer/assimp_importer_jni.cpp

```diff
@@ -30,6 +30,45 @@ Java_org_gearvrf_NativeAssimpImporter_getNumberOfMeshes(
 JNIEXPORT jlong JNICALL
 Java_org_gearvrf_NativeAssimpImporter_getMesh(JNIEnv * env,
         jobject obj, jlong jassimp_importer, jint index);
+JNIEXPORT jobject JNICALL
+Java_org_gearvrf_NativeAssimpImporter_getAssimpScene(JNIEnv * env,
+        jobject obj, jlong jassimp_importer);
+JNIEXPORT jlong JNICALL
+Java_org_gearvrf_NativeAssimpImporter_getNodeMesh(JNIEnv * env,
+        jobject obj, jlong jassimp_importer, jstring jnode_name, jint index);
+JNIEXPORT jobject JNICALL
+Java_org_gearvrf_NativeAssimpImporter_getMeshMaterial(JNIEnv * env,
+        jobject obj, jlong jassimp_importer, jstring jnode_name, jint index);
+JNIEXPORT jfloatArray JNICALL
+Java_org_gearvrf_NativeAssimpImporter_decomposeTransformationMatrix(JNIEnv * env,
+        jobject obj, jfloatArray jtransformation_matrix);
+JNIEXPORT jint JNICALL
+Java_org_util_jassimp_Jassimp_getVKeysize(JNIEnv *env,
+        jclass cls);
+JNIEXPORT jint JNICALL
+Java_org_util_jassimp_Jassimp_getQKeysize(JNIEnv *env,
+        jclass cls);
+JNIEXPORT jint JNICALL
+Java_org_util_jassimp_Jassimp_getV3Dsize(JNIEnv *env,
+        jclass cls);
+JNIEXPORT jint JNICALL
+Java_org_util_jassimp_Jassimp_getfloatsize(JNIEnv *env,
+        jclass cls);
+JNIEXPORT jint JNICALL
+Java_org_util_jassimp_Jassimp_getintsize(JNIEnv *env,
+        jclass cls);
+JNIEXPORT jint JNICALL
+Java_org_util_jassimp_Jassimp_getuintsize(JNIEnv *env,
+        jclass cls);
+JNIEXPORT jint JNICALL
+Java_org_util_jassimp_Jassimp_getdoublesize(JNIEnv *env,
+        jclass cls);
+JNIEXPORT jint JNICALL
+Java_org_util_jassimp_Jassimp_getlongsize(JNIEnv *env,
+        jclass cls);
+JNIEXPORT jstring JNICALL
+Java_org_util_jassimp_Jassimp_getErrorString(
+        JNIEnv *env, jclass cls);
 }
```

Comment:
It seems to me that you are adding JNI methods of Jassimp class into this file which I don't think is good to well organize the project. I think you'd better create another file like jassimp_jni.cpp and put these JNI calls into that file.